tests/backends/test_pyevm.py

```diff
@@ -41,88 +37,111 @@ def test_generate_custom_genesis_state(self):
-        assert genesis_params['gas_limit'] == param_overrides['gas_limit']
+        assert genesis_params["block_number"] == 0
+        assert genesis_params["gas_limit"] == param_overrides["gas_limit"]
 
         # Use the the staticmethod to generate custom genesis parameters
         genesis_params = PyEVMBackend._generate_genesis_params(param_overrides)
-        assert genesis_params['block_number'] == 0
-        assert genesis_params['gas_limit'] == param_overrides['gas_limit']
+        assert genesis_params["block_number"] == 0
+        assert genesis_params["gas_limit"] == param_overrides["gas_limit"]
 
         # Only existing default genesis parameter keys can be overridden
-        invalid_overrides = {'gato': 'con botas'}
+        invalid_overrides = {"gato": "con botas"}
         with pytest.raises(ValueError):
-            _invalid_genesis_params = PyEVMBackend._generate_genesis_params(overrides=invalid_overrides)
+            _invalid_genesis_params = PyEVMBackend._generate_genesis_params(
+                overrides=invalid_overrides
+            )
 
     def test_override_genesis_parameters(self):
 
         # Establish a custom gas limit
-        param_overrides = {'gas_limit': 4750000}
+        param_overrides = {"gas_limit": 4750000}
         block_one_gas_limit = 4745362
 
         # Initialize PyEVM backend with custom genesis parameters
-        genesis_params = PyEVMBackend._generate_genesis_params(overrides=param_overrides)
+        genesis_params = PyEVMBackend._generate_genesis_params(
+            overrides=param_overrides
+        )
         pyevm_backend = PyEVMBackend(genesis_parameters=genesis_params)
         genesis_block = pyevm_backend.get_block_by_number(0)
-        assert genesis_block['gas_limit'] == param_overrides['gas_limit']
+        assert genesis_block["gas_limit"] == param_overrides["gas_limit"]
         genesis_block = pyevm_backend.get_block_by_number(1)
-        assert genesis_block['gas_limit'] == block_one_gas_limit
+        assert genesis_block["gas_limit"] == block_one_gas_limit
 
         # Integrate with EthereumTester
         tester = EthereumTester(backend=pyevm_backend)
         genesis_block = tester.get_block_by_number(0)
-        assert genesis_block['gas_limit'] == param_overrides['gas_limit']
+        assert genesis_block["gas_limit"] == param_overrides["gas_limit"]
         genesis_block = tester.get_block_by_number(1)
-        assert genesis_block['gas_limit'] == block_one_gas_limit
+        assert genesis_block["gas_limit"] == block_one_gas_limit
+
+    def test_send_transaction_invalid_from(self, eth_tester):
+        accounts = eth_tester.get_accounts()
+        assert accounts, "No accounts available for transaction sending"
+
+        with pytest.raises(ValidationError) as excinfo:
+            self._send_and_check_transaction(
+                eth_tester, SIMPLE_TRANSACTION, ZERO_ADDRESS_HEX
+            )
+
+        assert "does this account exist?" in str(excinfo.value)
```

Comment:
You can remove this in favor of providing a `match` keyword to the `pytest.raises` call:

https://docs.pytest.org/en/latest/assert.html